lib/baseTest.py

```diff
@@ -132,21 +136,39 @@ def remove_test_file(self):
         if hasattr(self, "test_url_id"):
             self.target_helper.delete_target(self.test_url_id)
 
-    def load_configs(self):
+    def _platform_dep_settings_handler(self, config_value):
+        if self.current_platform_name in config_value:
+            if self.current_platform_ver in config_value[self.current_platform_name]:
+                platform_dep_variables = copy.deepcopy(config_value[self.current_platform_name][self.current_platform_ver])
+            else:
+                platform_dep_variables = copy.deepcopy(config_value[self.current_platform_name][''])
+            config_value.update(platform_dep_variables)
+            return platform_dep_variables
+        return {}
+
+    def set_configs(self, config_variable_name, config_value):
```

Comment:
It's a config setter though. The function is use to create or set a new config, and the config can be overwrite if the config created. How about ... `set_and_update_config()` then?